test/utils.jl

```diff
@@ -1,4 +1,22 @@
 @testset "Utils" begin
-  # TODO: readcoda
-  # TODO: compose
+  @testset "readcoda" begin
+    df = readcoda(joinpath(datadir,"juraset.csv"); codanames=[:Cd, :Cu, :Pb, :Co, :Cr, :Ni, :Zn]);
```

Comment:
With the proposed changes above, we could write `codanames=(:Cd, :Cu, :Pb, :Co, :Cr, :Ni, :Zn)`